test-build.sh

```diff
@@ -74,12 +74,6 @@ for version in "${versions[@]}"; do
   tag=$(get_tag "${version}")
   full_version=$(get_full_version "${version}")
 
-  # Required for chakracore
-  if [ -f "${version}/Dockerfile" ]; then
```

Comment:
Think I might have gotten this wrong, and it might need to get unwrapped instead of deleting